packages/manager/modules/web-paas/src/project.class.js

```diff
@@ -0,0 +1,24 @@
+import { STATUS } from './web-paas.constants';
+
+export default class Project {
+  constructor(project) {
+    Object.assign(this, project);
+    if (this.partnerProjectId) {
+      this.displayName = `${this.projectName} - ${this.partnerProjectId}`;
+    } else {
+      this.displayName = this.projectName;
+    }
```

Comment:
Maybe we can create a getter to handle this 
```suggestion
   get displayName() {
     return this.partnerProjectId ? `${this.projectName} - ${this.partnerProjectId}` : this.projectName
   }
```